src/providers/ipa/ipa_subdomains.c

```diff
@@ -434,6 +435,12 @@ static errno_t ipa_subdom_store(struct sss_domain_info *parent,
         goto done;
     }
 
+    ret = sysdb_attrs_get_string(attrs, IPA_CN, &dns);
```

Comment:
Hi,

I think it might be more clear to just copy `name` to make clear that it is the same for IPA (similar as with `realm` as upper-case version of `name`).

bye,
Sumit

Reply:
Fixed.